.github/workflows/publish-alpha.yml

```diff
@@ -0,0 +1,80 @@
+name: "(▶) Publish Alpha Version"
+on:
+  workflow_dispatch:
+jobs:
+  createAlphaVersion:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: "0"
+      - name: Read .nvmrc
+        run: echo "##[set-output name=NVMRC;]$(cat .nvmrc)"
+        id: nvm
+      - name: Use Node.js (.nvmrc)
+        uses: actions/setup-node@v1
+        with:
+          node-version: "${{ steps.nvm.outputs.NVMRC }}"
+      - name: Install Dependencies and Build Packages
+        run: make init
+      - name: Git Identity
+        run: |
+          git config --global user.name 'frontegg'
+          git config --global user.email 'frontegg@users.noreply.github.com'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Authenticate with Registry
+        run: |
+          yarn logout
+          echo "init-author-name=Frontegg LTD" > .npmrc
+          echo "init-author-email=hello@frontegg.com" >> .npmrc
+          echo "init-author-url=https://frontegg.com" >> .npmrc
+          echo "init-license=MIT" >> .npmrc
+          echo "always-auth=true" >> .npmrc
+          echo "registry=https://registry.npmjs.org" >> .npmrc
+          echo "_authToken=$NPM_TOKEN" >> .npmrc
+          echo "@frontegg:registry=https://registry.npmjs.org" >> .npmrc
+          echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> .npmrc
+          npm whoami
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_PUBLISH_TOKEN }}
+      - name: Commit Version UP
+        run: |
+          yarn lerna version patch --no-git-tag-version --yes --no-push --force-publish
+          yarn update-version
+          yarn prettier-hook
+      - name: "Set incremented version"
+        uses: actions/github-script@v6
+        id: 'incremented-version'
+        with:
+          result-encoding: string
+          script: |
+            const {default: fs} = await import('fs');
+            const {version} = JSON.parse(fs.readFileSync('./lerna.json', {encoding: "utf-8"}));
+            return version;
+      - name: Publish Pre-Release version to NPM
+        id: publish_pre_release_version
+        run: |
+          version=$(node -p 'require("./lerna.json").version')
+          echo "::set-output name=LIB_VERSION::${version}"
+          echo "Publishing DEV version - v${version}-alpha.${{ github.run_id }}"
+          make move-package-json-to-dist
+          make prerelease-version-upgrade-${version}-alpha.${{ github.run_id }}
+          make pretty
+          git add .
+          git commit -m "chore: prelrease version"
```

Comment:
remove

Reply:
NOT A BLOCKER: to be honest, now when I'm looking deeply, I don't see where we push the commit in the `publish-prerelease.yml` - can you check what the purpose of this commit? maybe we should remove it from the other pipeline